app/src/main/res/xml/pref_advanced_settings.xml

```diff
@@ -70,6 +70,12 @@
                 android:summary="@string/pref_summary_wear_logs_prefix"
                 android:textIsSelectable="true"
                 android:title="@string/pref_wear_logs_prefix" />
+            <CheckBoxPreference
```

Comment:
Sorry, I DO think they phone and watch treatment prefs should be unique.  I do not want to sync the Treatment preferences between the devices.